src/pages/gen2/start/quickstart/vite-react-app/index.mdx

```diff
@@ -0,0 +1,270 @@
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+This Quickstart guide will walk you through how to build a task list application with TypeScript, Vite and React. If you are new to these technologies, we recommend you go through the official [React](https://react.dev/learn/tutorial-tic-tac-toe), [Vite](https://vitejs.dev/guide/), and [TypeScript](https://www.typescriptlang.org/docs/handbook/typescript-from-scratch.html) tutorials first.
+
+import prerequisites from 'src/fragments/gen2/quickstart/prerequisites.mdx';
+
+<Fragments fragments={{ javascript: prerequisites, nextjs: prerequisites }} />
+
+## Create a project
+
+From your project directory, run the following command and respond to the prompts:
+
+```bash title="Terminal" showLineNumbers={false}
+npm create vite@latest
+✔ Project name: react-amplify-gen2
+✔ Select a framework: › React
+✔ Select a variant: › TypeScript
+```
+
+This creates a new React app in a directory called react-amplify-gen2. You can then run the following commands to switch into the new react-amplified directory and install the project dependencies:
+
+```bash title="Terminal" showLineNumbers={false}
+cd react-amplify-gen2
+npm install
+```
+
+## Create a backend
+
+The easiest way to get started with AWS Amplify is through npm with `create-amplify`.
+
+```bash title="Terminal" showLineNumbers={false}
+npm create amplify@latest
+```
+
+```console title="Terminal" showLineNumbers={false}
+? Where should we create your project? (.) # press enter
+```
+
+Running this command will scaffold a lightweight Amplify project in your current project with the following files:
+
+```text
+├── amplify/
+│   ├── auth/
+│   │   └── resource.ts
+│   ├── data/
+│   │   └── resource.ts
+│   ├── backend.ts
+│   └── package.json
+├── node_modules/
+├── .gitignore
+├── package-lock.json
+├── package.json
+└── tsconfig.json
+```
+
+## Start local dev server
+
+Let's start a local dev server for your app development. For the frontend, run `npm run dev` to spin up a `localhost` dev server.
+
+```bash title="Terminal" showLineNumbers={false}
+npm run dev
+```
+
+<Callout info>
+
+Now [configure your AWS account to use Amplify](/gen2/start/account-setup/).  Note: If you already have an AWS profile with credentials on your local machine, and you have configured the corresponding AWS profile with the **AmplifyBackendDeployFullAccess** permission policy, you can skip this step.
+
+</Callout>
+
+## Start Amplify sandbox
+
+For the backend, we're going to start a cloud sandbox environment. Amplify gives every developer a personal cloud sandbox environment that provides isolated development spaces to rapidly build, test, and iterate. When you're working with a team, each developer will have their own personal cloud sandbox. In a new terminal window, run the following command:
+
+```bash title="Terminal" showLineNumbers={false}
+npx amplify sandbox
+```
+<Callout warning>Do not use cloud sandbox environments in production.</Callout>
```

Comment:
does this render okay without a newline separator?

Reply:
It does but I'll insert a new line for the consistency

Reply:
done :)

https://github.com/aws-amplify/docs/pull/6870/commits/bd4a344cbc0e2612f747cd0271d9f506002040e1